### DISCUSSION TITLE
Using the library to create a Android Application

### DISCUSSION THREAD

**Discussion by YasiruRuwantha** [Ant-Brain/EfficientWord-Net#26] [Q&A]
Can I use this to create an Android application? if there are any android demo please mention, And what will be the result if i were to found like 50 samples for each keyword? I intend to add like 5 keywords.

**Comment by aman-17** [comment]
You can create it for Android applications, we do not have API for this but you can use our model as a library to integrate with your application. You need to look for how to integrate python libraries in Android Studio/Flutter.

**Reply by rjpikus** [reply]
@YasiruRuwantha I'm starting an android implementation, Yasir, did you have success? Any dead ends? Thanks aman and lazy coder for the heavy lifting.

**Comment by TheSeriousProgrammer** [comment]
The current model best model is based on resnet and exported to onnx, We are exploring to see on how it can be converted to tflite, which will allow us to easily deploy the same in virtually any device (as tflite libraries are present for all devices)

For android specific use case, you can consider using onnxruntime https://onnxruntime.ai/docs/tutorials/mobile/

**Reply by rjpikus** [reply]
have you ever heard of sphinx? it's pretty easy to get running with there demo, but it isn't extremely accurate. ant's is going to take more work, but probably more accurates
